python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -449,7 +450,26 @@ class RandomForestClassifier(BaseRandomForestModel,
         if self.random_state is None:
             seed_val = <uintptr_t>NULL
         else:
```

Comment:
Minor nitpick: Can use an `elif` here instead of having an else branch for all the other cases.